operator/mina_account/lib/src/merkle_verifier.rs

```diff
@@ -14,15 +14,16 @@ pub fn verify_merkle_proof(merkle_leaf: Fp, merkle_path: Vec<MerklePath>, merkle
             .enumerate()
             .fold(merkle_leaf, |accum, (depth, path)| {
                 let hashes = match path {
-                    MerklePath::Left(right) => [accum, *right],
-                    MerklePath::Right(left) => [*left, accum],
+                    MerkleNode::Left(right) => [accum, *right],
+                    MerkleNode::Right(left) => [*left, accum],
                 };
 
                 param.clear();
                 write!(&mut param, "MinaMklTree{:03}", depth).unwrap();
 
                 hash_with_kimchi(param.as_str(), &hashes)
             });
+    println!("{}", LedgerHash::from_fp(calculated_root));
```

Comment:
Is this `print` needed for debugging? We can remove it if not to reduce operator logs.

Reply:
good catch!